Makefile

```diff
@@ -465,9 +465,9 @@ cluster/cleanup/olm: cluster/cleanup
 
 .PHONY: cluster/cleanup/crds
 cluster/cleanup/crds:
-	@-oc delete crd grafanadashboards.integreatly.org
```

Comment:
are we using the grafanadashboards at all? Is this required?

Reply:
seems it's not correct version. This is file:
https://github.com/valerymo/integreatly-operator/blob/MGDAPI-6011/Makefile#L468-L470
![Screenshot from 2023-10-18 12-52-53](https://github.com/integr8ly/integreatly-operator/assets/56625811/169e6427-77d3-4bfe-bdb2-9399a12b771a)